config/prod/nf-osi-processor.yaml

```diff
@@ -0,0 +1,19 @@
+template_path: managed-ec2-v5.yaml
+stack_name: nf-osi-processor
```

Comment:
If the intention is to create a new instance then you must give it a unique name. This one already exists.

Reply:
It doesn't.  nf-osi-processing exists